common/common-rest/src/main/java/io/servicecomb/common/rest/codec/QueryTypeEnum.java

```diff
@@ -0,0 +1,37 @@
+package io.servicecomb.common.rest.codec;
+
+public enum QueryTypeEnum {
```

Comment:
why we need this enum?

Reply:
as collectionFormat has csv,multi,ssv and more than 5 type to impress , i think use enum to do it would be better to expand